ui-v2/src/api/flow-runs/index.ts

```diff
@@ -9,6 +9,9 @@ import { components } from "../prefect";
 import { getQueryService } from "../service";
 
 export type FlowRun = components["schemas"]["FlowRun"];
+export type FlowRunWithFlow = FlowRun & {
+	flow: Flow;
+};
```

Comment:
For context I believe this type is around mostly because the FlowRun model doesn't have `flow_name` but just `flow_id`. In cloud we recently added `flow_name` to the model and got rid of this type and the extra fetching to get the flow.

Reply:
idk that there's plan to do so but I'm sure we can make a plan to add it